### PR TITLE
Revert #8978

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

title

done on head maintainer's request. Was merged prematurely

image

## Changelog

:cl:
expansion: added back HoP's armour
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
Still remove the ability to have a gun in its suit slot. It's not needed.

File: modular_skyrat/modules/customization/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -79,6 +79,11 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	cold_protection = CHEST|GROIN|ARMS
 	heat_protection = CHEST|GROIN|ARMS
+	armor = list(MELEE = 25, BULLET = 20, LASER = 20, ENERGY = 30, BOMB = 20, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 8)
+
+/obj/item/clothing/suit/toggle/hop_parade/Initialize()
+	. = ..()
+	allowed = GLOB.security_wintercoat_allowed
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/suits/armor.dm:86]: No more of this. You choose between fashion and open carrying. You don't have both as a non-combat-oriented role.
```suggestion
```

File: modular_skyrat/modules/customization/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -79,6 +79,11 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	cold_protection = CHEST|GROIN|ARMS
 	heat_protection = CHEST|GROIN|ARMS
+	armor = list(MELEE = 25, BULLET = 20, LASER = 20, ENERGY = 30, BOMB = 20, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 8)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/suits/armor.dm:82]: Why does it have wound armor?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by jjpark-kb** [comment]
stale.

**Pull request closed without merging**